qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -127,8 +129,12 @@ def setUp(self):
             if ent_type == "client" and ent_id not in client_mount_ids and ent_id != "admin":
                 self.mds_cluster.mon_manager.raw_cluster_cmd("auth", "del", entry['entity'])
 
+        self.volname = None
+        self.vol_created = False
```

Comment:
I think it's sufficient to just test if `self.fs is None` later so I think we can remove the above two variables.

Reply:
self.volname is heavily used in 'TestVolumes' (test_volumes.py).

Reply:
Addressed in https://github.com/ceph/ceph/pull/36368